.editorconfig

```diff
@@ -7,3 +7,4 @@ insert_final_newline = true
 indent_style = space
 indent_size = 2
 trim_trailing_whitespace = true
+max_line_length = 120
```

Comment:
We don't really use that and it seems a bit arbitrary to add it for everyone.
We try to enforce code styling as little as possible.